go/kbfs/libkbfs/block_retrieval_queue.go

```diff
@@ -610,6 +611,8 @@ func (brq *blockRetrievalQueue) FinalizeRequest(
 		if err != nil {
 			brq.log.CDebugf(
 				retrieval.ctx, "Couldn't put block in cache: %+v", err)
+			// swallow the error if we were unable to put the block into caches.
```

Comment:
Do we want to do something else here eventually? If so, should we TODO and ticket?

Or are we planning to evict so this won't happen?

Reply:
I believe that it was never the intent to surface caching errors.